engine/dlib/src/test/test_socket.cpp

```diff
@@ -135,6 +135,9 @@ inline dmSocket::Socket GetSocket(dmSocket::Domain domain)
     result = dmSocket::SetNoDelay(instance, true);
     if (result != dmSocket::RESULT_OK) return -1;
 
+    result = dmSocket::SetQuickAck(instance, true);
```

Comment:
Should this be configurable perhaps and not always on? Are there disadvantages of having quick ACK always enabled? I'm thinking of usecase when the socket is used for an HTTP connection vs a TCP connection with a lot of data going back and forth (games)?

LuaSocket has a setoption() function with options such as "keepalive" and "tcp-nodelay" as valid choices.

Reply:
I only set it here in the tests to make sure it gets used.
In the engine itself we use whatever is the default.

Reply:
Or perhaps I'm missing your point :)

Reply:
Ah, I'm confused. It was a test :-)